test/upgrade/2.0.0-2.1.0.js

```diff
@@ -175,16 +194,561 @@ describe("[@skip-on-coverage] After facet upgrade, everything is still operation
+        expect(support, "Interface should be supported").is.true;
+      });
+
+      it("Supported interface can be removed", async function () {
+        const interfaceId = "0xddccbbaa";
+        // Add interface
+        await ERC165Facet.connect(deployer).addSupportedInterface(interfaceId);
+
+        // Verify that interface exist
+        let support = await ERC165Facet.supportsInterface(interfaceId);
+        expect(support, "Interface should be supported").is.true;
+
+        // Remove interface
+        await ERC165Facet.connect(deployer).removeSupportedInterface(interfaceId);
+
+        // Verify it was removed
+        support = await ERC165Facet.supportsInterface(interfaceId);
+        expect(support, "Interface should not be supported").is.false;
+      });
+
+      it("Seller can be updated in two steps", async function () {
+        const oldSeller = sellers[0][3];
+
+        const seller = oldSeller.seller.clone();
+        seller.treasury = treasury.address;
+        // Treasury is the only values that can be update without address owner authorization
+        let sellerStruct = seller.toStruct();
+
+        seller.admin = admin.address;
+        seller.operator = operator.address;
+        seller.clerk = clerk.address;
+
+        const pendingSellerUpdate = seller.clone();
+        pendingSellerUpdate.id = "0";
+        pendingSellerUpdate.treasury = ethers.constants.AddressZero;
+        pendingSellerUpdate.active = false;
+        let pendingSellerUpdateStruct = pendingSellerUpdate.toStruct();
+
+        const authToken = mockAuthToken();
+        const pendingAuthToken = authToken.clone();
+        const oldSellerAuthToken = oldSeller.authToken.toStruct();
+        const pendingAuthTokenStruct = pendingAuthToken.toStruct();
+
+        // Update seller
+        let tx = await accountHandler.connect(oldSeller.wallet).updateSeller(seller, authToken);
+
+        // Testing for the SellerUpdateApplied event
+        await expect(tx)
+          .to.emit(accountHandler, "SellerUpdateApplied")
+          .withArgs(
+            seller.id,
+            sellerStruct,
+            pendingSellerUpdateStruct,
+            oldSellerAuthToken,
+            pendingAuthTokenStruct,
+            oldSeller.wallet.address
+          );
+
+        // Testing for the SellerUpdatePending event
+        await expect(tx)
+          .to.emit(accountHandler, "SellerUpdatePending")
+          .withArgs(seller.id, pendingSellerUpdateStruct, pendingAuthTokenStruct, oldSeller.wallet.address);
+
+        // Update seller operator
+        tx = await accountHandler.connect(operator).optInToSellerUpdate(seller.id, [SellerUpdateFields.Operator]);
+
+        pendingSellerUpdate.operator = ethers.constants.AddressZero;
+        pendingSellerUpdateStruct = pendingSellerUpdate.toStruct();
+        seller.clerk = oldSeller.seller.clerk;
+        seller.admin = oldSeller.seller.admin;
+        sellerStruct = seller.toStruct();
```

Comment:
The operator update was just confirmed. Shouldn't seller.operator = seller.address at this point?

Reply:
`seller` is the new seller and already have the correct value in `operator` field.

To avoid confusion I added a new value - `expectedSeller`, so it's easier to follow the changes.